app/facades/garden_facade.rb

```diff
@@ -6,23 +6,23 @@ class GardenFacade < DatabaseService
 
   def self.get_garden(garden_id)    #get_garden_endpoint
     json = DatabaseService.get_garden_endpoint(garden_id)
-    Garden.new(json[:data][:attributes], json[:data][:id])
+    Garden.new(json[:data])
   end
 
   def self.get_garden_plots(garden_id)    #get_garden_plots_endpoint
     json = DatabaseService.get_garden_plots_endpoint(garden_id)
-    json[:data].map { |plot| GardenPlot.new(plot, garden_id) }
+    json[:data].map { |plot| Plot.new(plot, garden_id) }
   end
 
   def self.get_user_gardens(user_id)    #get_user_gardens_endpoint
     json = DatabaseService.get_user_gardens_endpoint(user_id)
-    json[:data].map { |garden| Garden.new(garden[:attributes], garden[:id])}
+    json[:data].map { |garden| GardenPoro.new(garden[:attributes], garden[:id])}
```

Comment:
It looks like the poros was renamed to Garden but here it is GardenPoro, should this be switched?